src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -208,14 +208,36 @@ class TableView extends React.Component<TableViewProps> {
     if (!tableData || !tableData.meta) {
       return dataRow[column.key];
     }
-    const fieldRenderer = getFieldRenderer(String(column.key), tableData.meta);
+
+    const columnKey = String(column.key);
+    const fieldRenderer = getFieldRenderer(columnKey, tableData.meta);
 
     const display = eventView.getDisplayMode();
     const isTopEvents =
       display === DisplayModes.TOP5 || display === DisplayModes.DAILYTOP5;
 
     const count = Math.min(tableData?.data?.length ?? TOP_N, TOP_N);
 
+    let cell = fieldRenderer(dataRow, {organization, location});
+
+    if (columnKey === 'id') {
+      const eventSlug = generateEventSlug(dataRow);
+
+      const target = eventDetailsRouteWithEventView({
+        orgSlug: organization.slug,
+        eventSlug,
+        eventView,
+      });
+
+      cell = (
+        <Tooltip key={`eventlink${rowIndex}`} title={t('View Event')}>
```

Comment:
Using row indexes as keys has burned me in the past. On re-renders you might get incorrect component states. Using the event id/slug would avoid that.

Reply:
Actually, now that I think about it, I don't need the key at all. I'm just going to remove it.